python/trezorlib/tests/device_tests/test_msg_recoverydevice_shamir.py

```diff
@@ -18,343 +18,219 @@
-            "extra extend academic arcade born dive legal hush gross briefing talent drug much home firefly toxic analysis idea umbrella slice",
-        ]
-        word_count = len(mnemonics[0].split(" "))
-
-        ret = self.client.call_raw(
-            proto.RecoveryDevice(
-                passphrase_protection=False, pin_protection=False, label="label"
-            )
-        )
-
-        # Confirm Recovery
-        assert isinstance(ret, proto.ButtonRequest)
-        self.client.debug.press_yes()
-        ret = self.client.call_raw(proto.ButtonAck())
-
-        # Homescreen - consider aborting process
-        assert isinstance(ret, proto.ButtonRequest)
-        self.client.debug.press_no()
-        ret = self.client.call_raw(proto.ButtonAck())
-
-        # Homescreen - but then bail out in the warning
-        assert isinstance(ret, proto.ButtonRequest)
-        self.client.debug.press_no()
-        ret = self.client.call_raw(proto.ButtonAck())
-
-        # Homescreen - click Enter
-        assert isinstance(ret, proto.ButtonRequest)
-        self.client.debug.press_yes()
-        ret = self.client.call_raw(proto.ButtonAck())
-
-        # Enter word count
-        assert ret == proto.ButtonRequest(
-            code=proto.ButtonRequestType.MnemonicWordCount
-        )
-        self.client.debug.input(str(word_count))
-        ret = self.client.call_raw(proto.ButtonAck())
-
-        # Homescreen
-        assert isinstance(ret, proto.ButtonRequest)
-        self.client.debug.press_yes()
-        ret = self.client.call_raw(proto.ButtonAck())
-
-        # Enter shares
-        for mnemonic in mnemonics:
-            # Enter mnemonic words
-            assert ret == proto.ButtonRequest(
-                code=proto.ButtonRequestType.MnemonicInput
-            )
-            self.client.transport.write(proto.ButtonAck())
-            for word in mnemonic.split(" "):
-                time.sleep(1)
-                self.client.debug.input(word)
-            ret = self.client.transport.read()
-
-            if mnemonic != mnemonics[-1]:
-                # Homescreen
-                assert isinstance(ret, proto.ButtonRequest)
-                self.client.debug.press_yes()
-                ret = self.client.call_raw(proto.ButtonAck())
-
-        # Confirm success
-        assert isinstance(ret, proto.ButtonRequest)
-        self.client.debug.press_yes()
-        ret = self.client.call_raw(proto.ButtonAck())
-
-        # Workflow succesfully ended
-        assert ret == proto.Success(message="Device recovered")
-
-        assert self.client.features.pin_protection is False
-        assert self.client.features.passphrase_protection is False
-
-        # Check mnemonic
-        assert (
-            self.client.debug.read_mnemonic_secret().hex()
-            == "491b795b80fc21ccdf466c0fbc98c8fc"
+from trezorlib import btc, messages, device, exceptions
+
+pytestmark = pytest.mark.skip_t1
```

Comment:
This is just to avoid marking all test cases?

Reply:
yes